seqr_scripts/make_remap_file.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import csv
+import logging
+from os import replace
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("Make remap file")
+logger.setLevel(logging.INFO)
+
+
+def read_file(file_path: str):
```

Comment:
```suggestion
def read_file(file_path: str) -> Iterator[Tuple[str, str]]:
```
you'll also need to import Iterator and Tuple (`from typing import Iterator, Tuple`)